clients/admin-ui/src/pages/_app.tsx

```diff
@@ -30,7 +30,7 @@ const MyApp = ({ Component, pageProps }: AppProps) => (
   <SafeHydrate>
     <Provider store={store}>
       <PersistGate loading={null} persistor={persistor}>
-        <ChakraProvider theme={theme}>
+        <FidesProvider theme={theme}>
```

Comment:
This will provide a default theme according to our theme package: https://github.com/ethyca/fidesui/blob/9e634242db1ffba6c54d4e047d4dfbd4bfbbd2d4/packages/react-theme/src/index.tsx#L4-L5
But we're overriding it anyway, so hopefully no difference.